apps/api/src/server/utils/document-storage.js

```diff
@@ -18,6 +18,7 @@ import config from '../config/config.js';
  * @typedef {DocumentInfo & WithBlobUrl} DocumentInfoWithBlobUrl
  */
 
+// TODO: DJW should we not use  * @returns {Promise<import('@pins/applications.api').Api.DocumentAndBlobInfoManyResponse>}
```

Comment:
Yes, makes sense to me! When I refactored this I tried to make as few changes as possible. Let's use that type, and remove the above type defs that we shouldn't need.